CHANGELOG.md

```diff
@@ -4,6 +4,223 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.27.0] - 2023-08-18
+
+## 🚀 Features
+
+### add a metric tracking coprocessor latency ([Issue #2924](https://github.com/apollographql/router/issues/2924))
+
+Introduces a new metric for the router:
+
+```
+apollo.router.operations.coprocessor.duration
+```
+
+It has one attributes:
+
+```
+coprocessor.stage: string (RouterRequest, RouterResponse, SubgraphRequest, SubgraphResponse)
+```
+
+It is an histogram metric tracking the time spent calling into the coprocessor
```

Comment:
```suggestion
It is an histogram metric tracking the time spent calling into the coprocessor.

Note that the name of this metric may change in the future.
```

Reply:
fixed in [00a819e](https://github.com/apollographql/router/pull/3602/commits/00a819eb1959c6f8f6dd9bcfab662267b5b5d29b)